src/main/java/org/frankframework/frankdoc/model/FrankAttribute.java

```diff
@@ -95,7 +95,7 @@ void handleDefaultExplicitNull(FrankType parameterType) {
 		}
 		boolean isExplicitNull = (StringUtils.isBlank(getDefaultValue()) || getDefaultValue().equals("null"));
 		if(isExplicitNull && parameterType.isPrimitive()) {
-			log.warn("Attribute [{}] is of primitive type [{}] but has default value null", toString(), parameterType.toString());
+			log.error("Attribute [{}] is of primitive type [{}] but has default value null", toString(), parameterType.toString());
```

Comment:
Deze moet WARN zijn.

Reply:
Zie PR https://github.com/ibissource/iaf/pull/2448. Als je die accepteert, dan treedt deze error niet meer op. Deze logging zegt dat je een default value null documenteert voor een setter van primitive type. Dat is tegenstrijdig. Je moet of een @ff.default documenteren die overeenkomt met de default, of je moet niets documenteren over de default.

Reply:
Fixed in https://github.com/ibissource/iaf/pull/2448